src/en/koharu/src/eu/kanade/tachiyomi/extension/en/koharu/Koharu.kt

```diff
@@ -47,19 +48,23 @@ class Koharu : HttpSource(), ConfigurableSource {
 
     private val json: Json by injectLazy()
 
+    private val B_REGEX = Regex("\\s*\\[.*?]\\s*|\\s*\\(.*?\\)\\s*")

```

Comment:
IIRC, you would run into less issues with escaping if you use a triple quoted string

Reply:
Should also be more greedy, otherwise it will fail on `123 (456 (789) abc)`. Or are you specifically doing that to avoid another issue?

Reply:
I mean- you should remove the lazy token (`?`). Currently it fails on `123 (456 (789) ab c)`.

I would go for something like:
```
\s*[([{].*[)\]}]\s*
```